test/e2etest/installer_test.go

```diff
@@ -56,6 +57,10 @@ func runInstallScript(configPath string) {
 
 	// wait `cmd` until it finishes
 	_ = cmd.Wait()
+
+	// run again with same config. This is to assure the `re-apply` action works without any issue.
+	_ = cmd.Start()
+	_ = cmd.Wait()
```

Comment:
Re apply test

Reply:
Did it run for a second time? I couldn't find it in the Github action..

Reply:
I just confirmed with manual run and it worked